modules/shared/api/postsApi.api.ts

```diff
@@ -0,0 +1,6 @@
+import { PostsApi } from '@m3ntorship/posts-client';
+import type { Configuration } from '@m3ntorship/posts-client/dist/client';
+
+export const postsApi = new PostsApi({
+  basePath: 'http://localhost:4010',
```

Comment:
for niw, this is ok. in the future, this should be a configuration not hard-coded